vm/src/vm.rs

```diff
@@ -197,14 +205,22 @@ impl VirtualMachine {
             vm.new_str("sys".to_owned()),
             vm.get_none(),
         );
+        if let Some(as_external) = initialize_parameter {
+            vm.initialize(as_external);
+        }
+        vm
+    }
+
+    pub fn initialize(&self, external: bool) {
```

Comment:
Could this method be private, to prevent double initialization from elsewhere?

Reply:
Thanks for the review.

In wasm, If the VM is initialized and then injected, these bugs will still there.
The VM should be injected before initialized, such that the `_browser`, `_js` can be correctly initialized.  so the `initialize` method is public.

The root cause is that the `builtin_module_names` is set in `make_module` according to the `stdlib_inits` in the timing of initializing.

Reply:
The `initialized` flag is now used to prevent double initialization.  Thanks for the view.